### PR TITLE
Always show trend graph tab.

### LINKED ISSUE #8825: Always show Trend graph tab
Use a loader until the graph is ready.

For metrics with the version number scale: show a message: "Trend graphs for metrics with the version number scale are not supported."

### PR DESCRIPTION
**Pull request by fniessink**
- Show warning message when the metric has a version number scale that trend graphs are not supported.
- Show warning message when the metric has no measurements yet.
- Show loader when loading measurements.
- Show warning message when loading failed.

Closes #8825.

### CODE REVIEW

File: components/frontend/src/metric/MetricDetails.js [hunk 1]
```diff
@@ -81,20 +81,23 @@ Buttons.propTypes = {
     url: string,
 }
 
-function fetchMeasurements(reportDate, metric_uuid, setMeasurements) {
+function fetchMeasurements(reportDate, metric_uuid, setMeasurements, setMeasurementsStatus) {
     get_metric_measurements(metric_uuid, reportDate)
         .then(function (json) {
-            if (json.ok !== false) {
-                setMeasurements(json.measurements)
-            }
```
Comment (fniessink) [inline components/frontend/src/metric/MetricDetails.js:89]: This if was never false: the API-server GET measurements endpoint does not have an ok flag.

File: docs/src/changelog.md [hunk 2]
```diff
@@ -8,6 +8,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
...
 
+## [Unreleased]
+
+### Deployment notes
+
+If your currently installed *Quality-time* version is not v5.14.0, please first check the upgrade path in the [versioning policy](versioning.md).
+
+### Changed
+
+- Always show metric trend graph tabs. Display a loader in the tab while loading measurements. If the trend graph cannot be displayed, explain in the tab why not. Possible reasons are: the metric has a version number scale (can't draw graphs for version numbers), the metric has no measurements yet, and loading the measurements failed. Closes [#8825](https://github.com/ICTU/quality-time/issues/8825).
```
Comment (wkoot) [inline docs/src/changelog.md]: This seems rather long for a changelog

File: components/frontend/src/metric/TrendGraph.js [hunk 3]
```diff
@@ -1,20 +1,55 @@
...
+        return (
+            <WarningMessage
+                content="Loading the measurements from the API-server failed."
+                header="Loading measurements failed"
+            />
+        )
+    }
+    if (loading === "loading") {
+        return (
+            <Placeholder fluid inverted={darkMode} style={{ height: chartHeight + 200 }}>
```
Comment (wkoot) [inline components/frontend/src/metric/TrendGraph.js]: Hmm, double magic number!

File: docs/src/changelog.md (hunk 2 above)
Comment (fniessink) [reply docs/src/changelog.md]: I'll remove the "Possible reasons..."

File: components/frontend/src/metric/TrendGraph.js (hunk 3 above)
Comment (fniessink) [reply components/frontend/src/metric/TrendGraph.js]: Yeah, the chart height does not include the height of the title and other decorations so I had to estimate that. I'll make it a constant so it's clearer why the placeholder needs to be higher than the chart height.

**Pull request merged**